webapp/channels/src/components/widgets/menu/menu_items/menu_start_trial.tsx

```diff
@@ -45,54 +49,42 @@ const MenuStartTrial = (props: Props): JSX.Element | null => {
             dialogType: LearnMoreTrialModal,
         }));
     };
-
-    const openTrialBenefitsModal = () => {
-        trackEvent(
-            TELEMETRY_CATEGORIES.SELF_HOSTED_START_TRIAL_MODAL,
-            'open_trial_benefits_modal_from_menu',
-        );
-        dispatch(openModal({
-            modalId: ModalIdentifiers.TRIAL_BENEFITS_MODAL,
-            dialogType: TrialBenefitsModal,
-        }));
-    };
-
     const prevTrialLicense = useSelector((state: GlobalState) => state.entities.admin.prevTrialLicense);
     const license = useSelector(getLicense);
     const isPrevLicensed = prevTrialLicense?.IsLicensed;
     const isCurrentLicensed = license?.IsLicensed;
-    const isCurrentLicenseTrial = isTrialLicense(license);
 
-    // Show this CTA if the instance is currently not licensed and has never had a trial license loaded before
-    const show = (isCurrentLicensed === 'false' && isPrevLicensed === 'false') || isCurrentLicenseTrial;
-    if (!show) {
+    if (isCurrentLicensed === 'true') {
         return null;
     }
 
+    const showTrialButton = isCurrentLicensed === 'false' && isPrevLicensed === 'false' && hasPermissionForStartTrial && isE0;
+
     return (
         <li
             className={'MenuStartTrial'}
             role='menuitem'
             id={props.id}
         >
-            {isCurrentLicenseTrial ? <>
-                <div style={{display: 'inline'}}>
-                    <span>
-                        {formatMessage({id: 'navbar_dropdown.reviewTrialBenefits', defaultMessage: 'Review the features you get with Enterprise. '})}
-                    </span>
-                    <button onClick={openTrialBenefitsModal}>
-                        {formatMessage({id: 'navbar_dropdown.learnMoreTrialBenefits', defaultMessage: 'Learn More'})}
-                    </button>
-                </div>
-            </> : <>
+            <div className='free_version_badge'>{'FREE EDITION'}</div>
```

Comment:
Is i18n needed here?

Reply:
It is intentionally not translatable. `FREE EDITION` here act as a name, not as regular text.